iOS/DuckDuckGo/DuckPlayer/NativeUI/DuckPlayerEntryPillView.swift

```diff
@@ -0,0 +1,132 @@
+//  limitations under the License.
+//
+
+import SwiftUI
+import DesignResourcesKit
+
+struct DuckPlayerEntryPillView: View {
+    @ObservedObject var viewModel: DuckPlayerEntryPillViewModel
+    
+    struct Constants {
+        static let daxLogo = "Home"
+        static let playImage = "play.fill"
+    }
+
+    private var sheetContent: some View {
+        Button(action: { viewModel.openInDuckPlayer()}) {
+            VStack(spacing: 12) {
+                HStack(spacing: 12) {
+                    Image(Constants.daxLogo)
+                        .resizable()
+                        .frame(width: 40, height: 40)
+                    
+                    Text(UserText.duckPlayerNativeOpenInDuckPlayer)
+                        .font(.system(size: 16, weight: .semibold))
+                        .foregroundColor(Color(designSystemColor: .textPrimary))
+                    
+                    Spacer()
+                    
+                    Image(systemName: Constants.playImage)
+                        .font(.system(size: 20))
+                        .foregroundColor(.white)
+                        .frame(width: 44, height: 44)
+                        .background(Color.blue)
+                        .clipShape(Circle())
+            
+                }
+                .padding(.horizontal, 16)
+                .padding(.vertical, 14)
+            }
+            .background(Color(designSystemColor: .surface))        
+            .cornerRadius(12)        
+            .shadow(color: Color.black.opacity(0.2), radius: 8, x: 0, y: 4)        
+            .padding(.horizontal, 16)
+            .padding(.vertical, 12)   
+            .padding(.bottom, 80)
+        }
+    }
+    
+    var body: some View {
+        GeometryReader { geometry in
+            VStack {
+                Spacer()
+                ZStack {
+                    // Background panel that extends beyond the bottom
+                    VStack(spacing: 0) {
+                        TopRoundedRectangle(radius: 12)
+                            .fill(Color(designSystemColor: .panel))
+                            .shadow(color: Color.black.opacity(0.08), radius: 8, y: -2)
+                        
+                        // Extra panel extension for bounce
+                        Rectangle()
+                            .fill(Color(designSystemColor: .panel))
+                            .frame(height: 60)
+                    }
+                    
+                    // Content
+                    VStack {
+                        sheetContent
+                        Spacer()
+                            .frame(height: 0)
+                    }
+                }
+            }
+            .offset(y: viewModel.isVisible ? 16 : geometry.size.height)
+        }
+        .edgesIgnoringSafeArea(.all)
+        .animation(viewModel.shouldAnimate ? .spring(duration: 0.4, bounce: 0.5, blendDuration: 1.0) : nil, value: viewModel.isVisible)
```

Comment:
Nit: Move the animation to a computed property

```swift
private var springAnimation: Animation? {
    viewModel.shouldAnimate ? .spring(duration: 0.4, bounce: 0.5, blendDuration: 1.0) : nil
}
```

then use like
`.animation(springAnimation, value: viewModel.isVisible)`